scripts/odbpy/io_place.py

```diff
@@ -35,52 +35,75 @@ def grid_to_tracks(origin, count, step):
     return tracks
 
 
-def equally_spaced_sequence(pin_count, possible_locations):
+def equally_spaced_sequence(side, side_pin_placement, possible_locations):
```

Comment:
Kindly read the contributors' guide. All Python code has to be formatted and linted before being merged into OpenLane. This file does not pass these checks after your modifications.